dspace-api/src/main/resources/org/dspace/storage/rdbms/sqlmigration/h2/V6.0_2017.07.04__DS-3636_General_Index_Performance_Updates.sql

```diff
@@ -0,0 +1,32 @@
+--
+-- The contents of this file are subject to the license and copyright
+-- detailed in the LICENSE and NOTICE files at the root of the source
+-- tree and available online at
+--
+-- http://www.dspace.org/license/
+--
+
+------------------------------------------------------
+-- DS-3636 General index and caching perfomance fixes
+------------------------------------------------------
+-- Indexes on resourcepolicy, versionitem, group2groupcache, metadata and bundle2bitstream colums to improve searching efficiency
+
+DROP INDEX IF EXISTS resourcepolicy_action_idx;
+DROP INDEX IF EXISTS resourcepolicy_resource_type_id_idx;
+DROP INDEX IF EXISTS resourcepolicy_resource_id_idx;
+DROP INDEX IF EXISTS versionitem_item_id_idx;
+DROP INDEX IF EXISTS versionitem_versionhistory_id_idx;
+DROP INDEX IF EXISTS group2groupcache_parent_id_idx;
+DROP INDEX IF EXISTS group2groupcache_child_id_idx;
+DROP INDEX IF EXISTS metadatavalue_mf_place_idx;
+DROP INDEX IF EXISTS bundle2bitstream_bitstream_order_idx;
+
+CREATE INDEX resourcepolicy_action_idx ON resourcepolicy (action_id);
+CREATE INDEX resourcepolicy_resource_type_id_idx ON resourcepolicy(resource_type_id );
+CREATE INDEX resourcepolicy_resource_id_idx ON resourcepolicy(resource_id );
+CREATE INDEX versionitem_item_id_idx ON versionitem (item_id);
+CREATE INDEX versionitem_versionhistory_id_idx ON versionitem (versionhistory_id);
+CREATE INDEX group2groupcache_parent_id_idx ON group2groupcache (parent_id);
+CREATE INDEX group2groupcache_child_id_idx ON group2groupcache (child_id);
```

Comment:
The above two indexes already exist in DSpace 6: https://github.com/DSpace/DSpace/blob/dspace-6.0/dspace-api/src/main/resources/org/dspace/storage/rdbms/sqlmigration/postgres/V6.0_2015.03.07__DS-2701_Hibernate_migration.sql#L206